coordinator/src/main/scala/com/socrata/datacoordinator/common/util/DatasetIdNormalizer.scala

```diff
@@ -0,0 +1,26 @@
+package com.socrata.datacoordinator.common.util
+
+import com.ibm.icu.text.Normalizer
+import com.rojoma.json.v3.io.{IdentifierEvent, FieldEvent, StringEvent, JsonEvent}
+
+object DatasetIdNormalizer {
+
+  val normalizationMode: Normalizer.Mode = Normalizer.NFC
```

Comment:
Its definitely unclear to me that we would use any other, and so unclear if we should make it configurable.